javascript/javascript1/week1/housey-pricey.js

```diff
@@ -0,0 +1,9 @@
+const peterHousevolume = 8 * 10 * 10;
+const juliaHouseVolume = 5 * 11 * 8;
+const peterGardenSize = 100;
+const juliaGardenSize = 70;
+const peterHousePrice = peterHousevolume * 2.5 * 1000 + peterGardenSize * 300;
+const juliaHousePrice = juliaHouseVolume * 2.5 * 1000 + juliaGardenSize * 300;
+
+console.log((peterHousePrice < 2500000) === true ? 'Peter is paying more' : 'peter is paying less');
```

Comment:
```suggestion
console.log((peterHousePrice < 2500000) ? 'Peter is paying more' : 'peter is paying less');
```
The expression always evaluates to true or false. you don't need to check

Reply:
Removed the comparison operator